### DISCUSSION TITLE
Wine

### DISCUSSION THREAD

**Discussion by achmadprayoogo** [elementary/appcenter#1952] [Q&A]
wine 5 cannot running windows software in elementary os 6.1
then, I tried to install wine 7, it always will be error in the repository. I founded "E: The repository 'http://ppa.launchpad.net/noobslab/apps/ubuntu focal Release' does not have a Release file." while tried to install the wine.

**Comment by lenemter** [comment]
https://launchpad.net/~noobslab/+archive/ubuntu/apps seems to be very old and unmaintained ppa that blocks update of all repositories. I suggest you removing it by executing `sudo rm /etc/apt/sources.list.d/noobslab-ubuntu-apps-focal.list`  in Terminal. And after that follow instructions at https://wiki.winehq.org/Ubuntu to install the latest Wine (elementary OS 6.1 is Ubuntu 20.04)
